content/influxdb/v2.0/reference/cli/influx/write/_index.md

```diff
@@ -23,45 +25,72 @@ Write data using [line protocol](/influxdb/v2.0/reference/syntax/line-protocol),
+|      | `--configs-path`    | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`)           | string      | `INFLUX_CONFIGS_PATH` |
+|      | `--debug`           | Output errors to stderr                                                         |             |                       |
+|      | `--encoding`        | Character encoding of input (default `UTF-8`)                                   | string      |                       |
+|      | `--error-file`      | Path to a file used for recording rejected row errors                           | string      |                       |
+| `-f` | `--file`            | File to import                                                                  | stringArray |                       |
+|      | `--format`          | Input format (`lp` or `csv`, default `lp`)                                      | string      |                       |
+|      | `--header`          | Prepend header line to CSV input data                                           | string      |                       |
+| `-h` | `--help`            | Help for the `write` command                                                    |             |                       |
+|      | `--host`            | HTTP address of InfluxDB (default `http://localhost:9999`)                      | string      | `INFLUX_HOST`         |
+|      | `--max-line-length` | Maximum number of bytes that can be read for a single line (default `16000000`) | integer     |                       |
+| `-o` | `--org`             | Organization name                                                               | string      | `INFLUX_ORG`          |
+|      | `--org-id`          | Organization ID                                                                 | string      | `INFLUX_ORG_ID`       |
+| `-p` | `--precision`       | Precision of the timestamps (default `ns`)                                      | string      | `INFLUX_PRECISION`    |
+|      | `--rate-limit`      | Throttle write rate (examples: `5 MB / 5 min` or `1MB/s`).                      | string      |                       |
+|      | `--skip-verify`     | Skip TLS certificate verification                                               |             |                       |
+|      | `--skipHeader`      | Skip first *n* rows of input data                                               | integer     |                       |
+|      | `--skipRowOnError`  | Output CSV errors to stderr, but continue processing                            |             |                       |
+| `-t` | `--token`           | Authentication token                                                            | string      | `INFLUX_TOKEN`        |
+| `-u` | `--url`             | URL to import data from                                                         | stringArray |                       |
 
 ## Examples
```

Comment:
Do we want to add an example to highlight diffs in extended annotated CSV via stdin?